_docs/install/health.md

```diff
@@ -63,7 +61,7 @@ By using this product, you are agreeing to the terms of the StorageOS Ltd. End U
 
 For plugin installs (`storageos/plugin`), use `journalctl` to view logs:
 ```bash
-$ journalctl | grep StorageOS
+$ journalctl -u docker | grep StorageOS
```

Comment:
This one is harder - to get only the StorageOS logs you need to find the plugin id and then grep on it.  Something like:
```
$ docker plugin ls
ID                  NAME                      DESCRIPTION                   ENABLED
fb4223538c46        storageos/plugin:latest   StorageOS plugin for Docker   false

$ sudo journalctl -u docker | grep fb4223538c46
Aug 16 15:34:37 storageos-1-38631 docker[606]: time="2017-08-16T15:34:37Z" level=info msg="By using this product, you are agreeing to the terms of the StorageOS Ltd. End User Subscription Agreement (EUSA) found at: https://storageos.com/legal/#eusa" plugin=fb4223538c46dbc7252c2dec210cb3fb6ac04693ee45ff322231ba34d9dc2ed7
Aug 16 15:34:37 storageos-1-38631 docker[606]: time="2017-08-16T15:34:37Z" level=info msg="==> Starting StorageOS server..." plugin=fb4223538c46dbc7252c2dec210cb3fb6ac04693ee45ff322231ba34d9dc2ed7
```